Decisions.Equifax/PrequalificationOfOne/Response/PreQualificationOfOneResponse.cs

```diff
@@ -0,0 +1,611 @@
+﻿using System.Runtime.Serialization;
+using DecisionsFramework.Design.ConfigurationStorage.Attributes;
+using Newtonsoft.Json;
+
+namespace Decisions.Equifax.PreQualificationOfOne.Response
+{
+  
+        [DataContract]
+        [Writable]
+        public class PreQualificationOfOneResponse
+        {
+            [JsonProperty("status")] public string Status { get; set; }
+
+            [JsonProperty("consumers")] public Consumers Consumers { get; set; }
+        }
+
+        [DataContract]
+        [Writable]
+        public partial class Consumers
+        {
+            [JsonProperty("equifaxUSConsumerCreditReport")]
+            public virtual EquifaxUsConsumerCreditReport[] EquifaxUsConsumerCreditReport { get; set; }
+        }
+
+
+        [DataContract]
+        [Writable]
+        public class EquifaxUsConsumerCreditReport
+        {
+            [JsonProperty("identifier")] public string Identifier { get; set; }
+
+            [JsonProperty("customerReferenceNumber")]
+            public string CustomerReferenceNumber { get; set; }
+
+            [JsonProperty("customerNumber")] public string CustomerNumber { get; set; }
+
+            [JsonProperty("consumerReferralCode")] public long ConsumerReferralCode { get; set; }
+
+            [JsonProperty("multipleReportIndicator")]
+            public string MultipleReportIndicator { get; set; }
+
+            [JsonProperty("ECOAInquiryType")] public string EcoaInquiryType { get; set; }
+
+            [JsonProperty("hitCode")]
+            [JsonConverter(typeof(FraudVictimIndicatorDataConverter))]
```

Comment:
Do we need this converter since we aren't sharing a base class between the two responses? Not sure how many parts there were to the issue

Reply:
I believe we still do, since Equifax admits that their sample data is incorrect, there is no guarantee that what is returned is 100 % accurate. I believe its best to play it safe